profile/reference-implementation.md

```diff
@@ -77,13 +79,13 @@ This section provides an overview of the key repositories of the EUDI Reference
 | [mDoc Data Model (iOS)](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-iso18013-data-model)    | Implementation of the mDoc data-model according to ISO/IEC 18013-5.           | 
 
 
-### Proximity Sharing Android Libraries
+### $\color{green}{\text{Proximity Sharing Android Libraries}}$
 
 | Repository          | Description                                                                   |
 |---------------|-------------------------------------------------------------------------------|
 | [mDoc Data Transfer (Android)](https://github.com/eu-digital-identity-wallet/eudi-lib-android-iso18013-data-transfer) | This library provides a set of classes to manage the transfer of documents in an EUDI ISO 18013-5 Android Wallet. |
 
-### Remote Presentation iOS Libraries
+### $\color{grey}{\text{Remote Presentation iOS Libraries}}$

```

Comment:
I recommend adopting a more standardized method for communicating the target/available platforms.

The recent changes have resulted in:
1. Inconsistencies in font family
2. Intrusive scrollbars
3. Increased cognitive load (subjective viewpoint)

![image](https://github.com/user-attachments/assets/d19c079e-9a90-4bb4-85a1-410db3d9e6ca)